chef-config/lib/chef-config/workstation_config_loader.rb

```diff
@@ -138,6 +146,34 @@ def working_directory
       a
     end
 
+    def apply_credentials(creds, _profile)
```

Comment:
Is accepting the profile here just future-proofing? Or am I missing some context (totally reasonable assumption on your part).